src/app/components/toast/index.html

```diff
@@ -61,4 +61,15 @@
       demoName="Toast">
     </sky-demo-page-code>
   </sky-demo-page-example>
+	
+	<sky-demo-page-content sectionHeading="UX guidelines">
+    <div class="sky-subsection-heading">Usage</div>
+		<p>Toasts are appropriate when:</p>
+		<ul>
+			<li>The trigger is server-side, indicating that a state in the system has changed.</li>
```

Comment:
I'd delete the comma and change "indicating" to "and indicates." ... And if it doesn't change the meaning, I'd change "that a state in the system has changed" to "a change in the system state." ... So: "The trigger is server-side and indicates a change in the system state."